apps/crn-frontend/src/network/teams/TeamOutput.tsx

```diff
@@ -77,7 +84,7 @@ const TeamOutput: React.FC<TeamOutputProps> = ({ teamId }) => {
   if (canCreateUpdate && team) {
     return (
       <Frame title="Share Research Output">
-        <TeamCreateOutputPage
+        <ResearchOutputPage
```

Comment:
I think this should be <ResearchOutputFormPage>

It would not be obvious to me that this wasn't the details page